types/types.proto

```diff
@@ -65,7 +65,7 @@ message ExecutionPayload {
   uint64 gasLimit = 8;
   uint64 gasUsed = 9;
   uint64 timestamp = 10;
-  types.H256 extraData = 11;
+  bytes extraData = 11;
```

Comment:
Why is `ExecutionPayload` in `types.proto` while other [Engine API](https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md) types like EngineForkChoiceUpdated are in ethbackend.proto? It seems to me that `types.proto` is reserved for more basic stuff and we should move `ExecutionPayload` to  `ethbackend.proto`.

Reply:
My reasoning is that, that is not really a neither a request or a reply and can works both way depending on what engine_ method you are using. so I put it in types because its behaviour more resembled a datatype rather than the request/reply of one single method.

Reply:
Hmm. OK. @AskAlexSharov what do you think?